Emby.Server.Implementations/Dto/DtoService.cs

```diff
@@ -898,7 +898,14 @@ private void AttachBasicFields(BaseItemDto dto, BaseItem item, BaseItem owner, D
                 dto.IsPlaceHolder = supportsPlaceHolders.IsPlaceHolder;
             }
 
-            dto.LUFS = item.LUFS;
+            if (item.LUFS.HasValue)
+            {
+                dto.NormalizationGain = -18f - item.LUFS;
```

Comment:
Maybe it worth to move `DefaultLUFSValue` const you have removed here? Or leave a comment about magic value (`-18f`)?